CRM/Price/BAO/LineItem.php

```diff
@@ -537,26 +541,4 @@ public static function getLineItemArray(&$params, $entityId = NULL, $entityTable
     }
   }
 
-  /**
-   * Calculate tax rate in percentage.
-   *
-   * @param array $lineItemId
-   *   An assoc array of lineItem.
-   *
-   * @return int|void
-   *   tax rate
-   */
-  public static function calculateTaxRate($lineItemId) {
```

Comment:
This function needs to be removed - it is fundamentally wrong to calculate Tax Rate using Tax Amount - which was calculated using Tax Rate.